packages/block-library/src/blocks/supernova-item/edit.js

```diff
@@ -14,12 +15,16 @@ import { MediaCompositionPreview } from "@novablocks/media-composition";
 
 import { SuperNova } from "@novablocks/block-editor";
 
-const { withPreviewAttributes } = SuperNova.utils;
 const { Card, CardButton, CardMediaWrapper } = SuperNova.components;
 
 const SuperNovaItemEdit = props => {
 
-  const blockProps = useBlockProps();
+  const blockProps = useBlockProps( {
+    className: classnames(
+      props.className,
+      'supernova-item'
+    )
+  } );
```

Comment:
It looks like in the frontend, we are using `supernova-collection__layout-item`. We need to choose one and be consistent.